src/AspNetCoreCustom/Example/public/index.html

```diff
@@ -1,12 +1,78 @@
+<!--
```

Comment:
Copied from server?

Reply:
Copied from their example repo. Edited the url of the license file and GraphQL path